wasm_src/ast_wrapper/ast_wrapper.cc

```diff
@@ -260,23 +260,6 @@ EMSCRIPTEN_KEEPALIVE int plotGrid(AstFrameSet* wcsinfo, double imageX1, double i
     double pbox[] = {imageX1, imageY1, imageX2, imageY2};
     plot = astPlot(wcsinfo, gbox, pbox, args);
 
-    // add RA/Dec reference
-    if (strlen(system) > 0)
-    {
-        const char* symbol1 = astGetC(plot, "Symbol(1)");
-        if (strcmp(symbol1, "RA") == 0 || strcmp(symbol1, "Dec") == 0)
```

Comment:
We check for "RA" and "Dec" when adding the reference. Please add the same logic in `OverlayStore`.